test/OntologyMockTrait.php

```diff
@@ -83,8 +86,10 @@ private function setupOntology(Ontology $onto)
             EventManager::SERVICE_ID => new EventManager(),
             LoggerService::SERVICE_ID => $this->prophesize(LoggerInterface::class)->reveal(),
             UriProvider::SERVICE_ID => new Bin2HexUriProvider([Bin2HexUriProvider::OPTION_NAMESPACE => 'http://ontology.mock/bin2hex#']),
-            SimpleCache::SERVICE_ID => new NoCache()
+            SimpleCache::SERVICE_ID => new NoCache(),
+            EventAggregator::SERVICE_ID => $eventAggregator
         ]);
+        $eventAggregator->setServiceLocator($sl);
```

Comment:
I know the name was already there, but would rename `$sl` and `$pm` to meaningful names